wcfsetup/install/files/lib/system/user/multifactor/totp/CodeFormField.class.php

```diff
@@ -32,6 +32,7 @@ public function __construct() {
 		$this->addFieldClass('multifactorTotpCode');
 		$this->autoComplete('off');
 		$this->inputMode('numeric');
+		$this->pattern('[0-9\s]*');
```

Comment:
```suggestion
		$this->pattern('[0-9]*');
```
TOTP / Email does not allow for spaces. Only backup codes do (because they are very long and shown in chunks).